en/ros/README.md

```diff
@@ -1,3 +1,5 @@
 # Robotics using ROS
 
-This section contains topics about using ROS (and offboard control) with PX4.
\ No newline at end of file
+[ROS](http://www.ros.org/) (Robot Operating System) is a general purpose robotics library that can be used with PX4 for offboard control. It uses the [MAVROS](../ros/mavros_installation.md) node to communicate with PX4 via MAVLink.
```

Comment:
FYI This is just because I think a section should properly introduce its content :-). 

@TSC21 is MAVROS required to communicate with PX4 in ROS1 ? Can you use some other comms node?
ie My understanding is that it is needed, whether you are going over wifi or via a uart. Also that ROS2 will support both MAVROS and an approach based on RTPS.

Reply:
There are other bridges, but the current "official" supported bridge between ROS and the MAVLink protocol is MAVROS. We are extending the support for fast-RTPS (using CDR serialization and deserialization) for the ROS1 implementation. There's currently an open project on the MAVROS repo for supporting ROS2 also - even though ROS2 supports RTPS by default, you need some kind of translation from the uORB msg to the common ROS world, and that's what we are working on on MAVROS.

Reply:
Thanks. I have captured this by adding a note to MAVROS doc about future support for RTPS, and removing the unnecessary clarification here that the communication is "via MAVLink". You can see the fix here: https://github.com/PX4/Devguide/pull/393/commits/6fa0b6f18972e9695451798a78361e74cb481f35